crates/tests/src/integration/masp.rs

```diff
@@ -1397,6 +1416,27 @@ fn masp_incentives() -> Result<()> {
     assert!(captured.result.is_ok());
     assert!(captured.contains("nam: 0.063"));
 
+    // Assert the rewards estimate is a number higher than the actual rewards
```

Comment:
what does this comment mean?

Reply:
Just a sanity check. The total amount of rewards issued for an asset should increase with each epoch. So estimations should reflect that.